### PR TITLE
core: fix filing test after v1.13.15 merge

### PR DESCRIPTION
**Pull request by AnnaShaleva**
This PR fixes failing tests after the v1.13.15 merge, the target branch is #232.

@txhsl, @chenquanyu, @songb2, welcome to review and test.

### CODE REVIEW

File: eth/handler_eth_test.go [hunk 1]
```diff
@@ -444,7 +444,6 @@ func testTransactionPropagation(t *testing.T, protocol uint) {
 
 // Tests that transactions get propagated to attached peers, either via
 // reannouncements/retrievals.
-func TestTransactionReannounce67(t *testing.T) { testTransactionReannounce(t, eth.ETH67) }
```
Comment (txhsl) [inline eth/handler_eth_test.go:447]: Glad to see this. :)

File: core/state_transition.go [hunk 2]
```diff
@@ -297,7 +297,7 @@ func (st *StateTransition) preCheck() error {
 		// Ensure the transaction is allowed by policy
 		// Apply policy minimum gas tip cap
 		var minGasTipCap = st.state.GetState(systemcontracts.PolicyProxyHash, systemcontracts.GetMinGasTipCapStateHash())
-		if msg.GasTipCap.Cmp(minGasTipCap.Big()) < 0 {
+		if msg.GasTipCap != nil && msg.GasTipCap.Cmp(minGasTipCap.Big()) < 0 {
```
Comment (roman-khimov) [inline core/state_transition.go]: Isn't it `(msg.GasTipCap == nil && minGasTipCap.Big().Sign() == 1) || (msg.GasTipCap != nil && msg.GasTipCap.Cmp(minGasTipCap.Big()) < 0)`? I'd say that a `nil` GasTipCap is similar to 0 if we have it for whatever reason.

Comment (AnnaShaleva) [reply core/state_transition.go]: Agree, fixed.

**Comment by AnnaShaleva** [comment]
Tests pass.

**Pull request merged**